lib/slax_web/websockets/token.ex

```diff
@@ -0,0 +1,288 @@
+defmodule SlaxWeb.Token do
+  alias Slax.{Projects, ProjectRepos, Slack}
```

Comment:
I think a `@moduledoc` here might help explain what this is for... bonus points if there are links to Slack docs on modals as well